keyboards/afternoonlabs/breeze/keymaps/summer/keymap.c

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 eithanshavit
```

Comment:
Similar question, maybe we can consider adding this to the default keymap instead.